### PR TITLE
feat: migration notes

### PR DESCRIPTION
**Pull request by burakukula**
Migration notes for tokens. WDYT?

### CODE REVIEW

File: packages/forma-36-tokens/MIGRATION.md
Comment (denkristoffer) [inline packages/forma-36-tokens/MIGRATION.md:6]:
```suggestion
The new approach allows us to meet all accessibility requirements. It also introduces a more consistent naming convention and the ability to easily scale the palette in the future if needed.
```

Not sure if a11y is an established name or just a convention?

File: packages/forma-36-tokens/MIGRATION.md
Comment (denkristoffer) [inline packages/forma-36-tokens/MIGRATION.md:36]: Maybe it makes more sense to show how the old tokens map to new ones here, rather than the values?

File: packages/forma-36-tokens/MIGRATION.md
Comment (denkristoffer) [inline packages/forma-36-tokens/MIGRATION.md:40]:
```suggestion
We took the same approach with all our color tokens, so now old and new tokens are available to you, with the same, consistent values.
```

Comment (denkristoffer) [reply packages/forma-36-tokens/MIGRATION.md:40]: > so now old and new tokens are available to you, with the same, consistent values.

Not sure what is meant by "consistent" here.

File: packages/forma-36-tokens/MIGRATION.md
Comment (denkristoffer) [inline packages/forma-36-tokens/MIGRATION.md:42]: > We plan to remove the old, deprecated tokend in the next major version

Do we?

File: packages/forma-36-tokens/MIGRATION.md
Comment (denkristoffer) [inline packages/forma-36-tokens/MIGRATION.md:52]: > see available transforms below.

This information is missing. But I'd suggest we just show the full command for transforming colors and remove this line completely.

**Review by denkristoffer — CHANGES_REQUESTED** [review]

File: packages/forma-36-tokens/MIGRATION.md
Comment (burakukula) [reply packages/forma-36-tokens/MIGRATION.md:40]: yeah, maybe it's not clear. I will rephrase it.

File: packages/forma-36-tokens/MIGRATION.md
Comment (burakukula) [reply packages/forma-36-tokens/MIGRATION.md:42]: I think we could plan for it, since we have a codemod? But dunno, let's check with the team.

File: packages/forma-36-tokens/MIGRATION.md
Comment (burakukula) [reply packages/forma-36-tokens/MIGRATION.md:66]: How about something like that? WDYT?

Comment (denkristoffer) [reply packages/forma-36-tokens/MIGRATION.md:66]: Works for me, thanks!

File: packages/forma-36-tokens/MIGRATION.md
Comment (denkristoffer) [reply packages/forma-36-tokens/MIGRATION.md:71]: If we don't get a timely answer I think the document works without this line too.

**Comment by denkristoffer** [comment]
Looks good, but I think we should decide on the line about removing tokens in v4 before we merge.

File: packages/forma-36-tokens/MIGRATION.md
Comment (thebesson) [reply packages/forma-36-tokens/MIGRATION.md:71]: we can change it to "we can provide you with tools to migrate to the new names"?

File: packages/forma-36-tokens/MIGRATION.md
Comment (denkristoffer) [inline packages/forma-36-tokens/MIGRATION.md:71]:
```suggestion
**Note: We plan to remove the old, deprecated tokens in a future breaking change of Forma 36, but don't worry, we provide you with tools to migrate to the new names.**
```

**Review by denkristoffer — CHANGES_REQUESTED** [review]

**Review by gui-santos — APPROVED** [review]
Loved it!

I +1 @denkristoffer suggestion on using "a future breaking change" in the text

**Review by denkristoffer — APPROVED** [review]

**Pull request merged**
